### PR TITLE
refactor(service): use env variables for configuration

### PR DESCRIPTION
**Pull request by samouss**
This PR updates how the service reads the configuration. It was read from a "database" file that wasn't tracked with `git`. The deployment was tied to the history of the production branch which is not flexible.

Now we have a new environment variable called `HELP_SCOUT_MAILBOXES` which the configuration stringified. The process to add a configuration is straight forward: get the value, parse it, update it, encode it and finally restart the service.

To make this process easier to implement/read I've extracted the logic of the service into its own module. We have a new `main` module which does only the plumbing between the dependencies e.g. read/parse environment variable, create the HelpScout client, ... At the end it makes the service easier to test.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { createHelpScoutClient } from './helpScoutClient';
+import { createService } from './createService';
+
+const githubWebhookSecret = process.env.GITHUB_WEBHOOK_SECRET || 'mySuperSecretToken';
+
+const helpScoutAppId = process.env.HELP_SCOUT_APP_ID || '';
+const helpScoutAppSecret = process.env.HELP_SCOUT_APP_SECRET || '';
+
+const helpScoutMailboxes = (process.env.HELP_SCOUT_MAILBOXES &&
+  JSON.parse(process.env.HELP_SCOUT_MAILBOXES)) || {
```
Comment (Haroenv) [inline src/main.ts]: json parse will error here, so fallback is never called

Comment (samouss) [reply src/main.ts]: Fallback is used when the value is not provided, if provided then we parse.

**Comment by samouss** [comment]
We have to test the branch before the merge. Test with/without `assignTo` attribute.

**Comment by samouss** [comment]
I've tested both usage with/without assignee, it looks like they both work.

**Pull request merged**